src/components/RequestDetails/SeeRequest.tsx

```diff
@@ -45,9 +46,11 @@ export const SeeRequest = ({ route: { params: p } }: RequestsNavigationProps<'SE
         </Text>
         <Box paddingRight="xl" />
       </ModalHeader>
-      <Box marginTop="l" paddingBottom="m" flex={1}>
-        <RequestDetails {...p} showStatus wasSent />
-      </Box>
+      <GestureRecognizer onSwipeRight={goBack} onEnded androidOnly>
+        <Box marginTop="l" paddingBottom="m" flex={1}>
```

Comment:
you can add style prop to gesture recognizer so we won't have to add extra box here